flexget/plugins/daemon/http_server.py

```diff
@@ -0,0 +1,92 @@
+from __future__ import unicode_literals, division, absolute_import
+import logging
+import threading
+
+from flexget.config_schema import register_config_key
+from flexget.event import event
+from flexget.utils.tools import singleton
+
+log = logging.getLogger('webui')
+
+main_schema = {
+    'type': 'object',
+    'properties': {
+        'bind': {'type': 'string', 'format': 'ipv4', 'default': '0.0.0.0'},
+        'port': {'type': 'integer', 'default': 5050},
+        'autoreload': {'type': 'boolean', 'default': False},
+        'authentication': {
```

Comment:
yes I agree. I left it as a plugin for now until we make a decision on if the api is baked in.

If it's baked in then there should be an option to point to a register an alternative webui...?